osf_tests/settings.py

```diff
@@ -18,3 +18,58 @@
 PASSWORD_HASHERS = (
     'django.contrib.auth.hashers.MD5PasswordHasher',
 )
+
+REST_FRAMEWORK = {
+    'PAGE_SIZE': 10,
+    # Order is important here because of a bug in rest_framework_swagger. For now,
+    # rest_framework.renderers.JSONRenderer needs to be first, at least until
+    # https://github.com/marcgibbons/django-rest-swagger/issues/271 is resolved.
+    'DEFAULT_RENDERER_CLASSES': (
+        'api.base.renderers.JSONAPIRenderer',
+        'api.base.renderers.JSONRendererWithESISupport',
+        'api.base.renderers.BrowsableAPIRendererNoForms',
+    ),
+    'DEFAULT_PARSER_CLASSES': (
+        'api.base.parsers.JSONAPIParser',
+        'api.base.parsers.JSONAPIParserForRegularJSON',
+        'rest_framework.parsers.FormParser',
+        'rest_framework.parsers.MultiPartParser'
+    ),
+    'EXCEPTION_HANDLER': 'api.base.exceptions.json_api_exception_handler',
+    'DEFAULT_CONTENT_NEGOTIATION_CLASS': 'api.base.content_negotiation.JSONAPIContentNegotiation',
+    'DEFAULT_VERSIONING_CLASS': 'api.base.versioning.BaseVersioning',
+    'DEFAULT_VERSION': '2.0',
+    # The versions below are specifically for testing purposes and do not reflect the actual versioning of the API.
+    # If changes are made to this list, or to DEFAULT_VERSION above, please reflect those changes in
+    # api_tests/base/test_versioning.py so that local tests will pass.
+    'ALLOWED_VERSIONS': (
```

Comment:
This is already set in `local-travis.py` here: https://github.com/CenterForOpenScience/osf.io/blob/2c4b8255df41f93374081f4019a6c030676dfecf/api/base/settings/local-travis.py#L28-L35 , so I'm not sure why this is necessary.